qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/DeleteNoExporterTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.broker.it.clustering;
+
+import io.zeebe.broker.it.GrpcClientRule;
+import io.zeebe.broker.system.configuration.BrokerCfg;
+import io.zeebe.broker.system.configuration.DataCfg;
+import java.util.Collections;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.RuleChain;
+
+public class DeleteNoExporterTest extends DeletionTest {
+  public static final int SNAPSHOT_PERIOD_SECONDS = 30;
+  public static final int MAX_SNAPSHOTS = 1;
+
+  public ClusteringRule clusteringRule = new ClusteringRule(1, 3, 3, this::configureNoExporters);
+  public GrpcClientRule clientRule = new GrpcClientRule(clusteringRule);
+
+  @Rule public RuleChain ruleChain = RuleChain.outerRule(clusteringRule).around(clientRule);
+
+  @Rule public ExpectedException expectedException = ExpectedException.none();
+
+  private void configureNoExporters(final BrokerCfg brokerCfg) {
+    final DataCfg data = brokerCfg.getData();
+    data.setMaxSnapshots(MAX_SNAPSHOTS);
+    data.setSnapshotPeriod(SNAPSHOT_PERIOD_SECONDS + "s");
+    data.setDefaultLogSegmentSize("8k");
+    data.setIndexBlockSize("2K");
+
+    brokerCfg.setExporters(Collections.EMPTY_LIST);
+  }
+
+  @Test
+  public void shouldDeleteDataOnFollowers() {
+    shouldDeleteDataOnFollowers(clusteringRule, clientRule);
```

Comment:
Please dont use inheritance in tests :see_no_evil: I have no clue what happens here